practice-two/src/js/script.js

```diff
@@ -1 +1,108 @@
+// Query to id
+const fullNameInput = document.getElementById('fullName');
+const emailInput = document.getElementById('email');
+const salaryInput = document.getElementById('salary');
+const cityInput = document.getElementById('city');
 
+// Check the input condition
```

Comment:
Wrong comment, should be what is it and what it used for. Ex Regex for validating a value/text format